src/views/composer.observe.js

```diff
@@ -148,6 +156,12 @@
               } else {
                 composer.selection.setAfter(prevNode);
               }
+            } else {
```

Comment:
I'm a bit confused about when does this case happen when previous node that was present somehow disappears in the process? Also the code seems to expect that aNode (LI element) that should have been be removed previously on line:

    domNode(aNode).transferContentTo(prevNode, true);

is somehow still present?
Is it some IE weirdness? How to test?